src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+export const MovieCard: React.FC<Movie> = (movie) => {
```

Comment:
Add proper props type:
```tsx
type MovieProps= {
  movie: Movie;
};
```

And destructure it properly:
```tsx
export const MovieCard: React.FC<MovieProps> = ({ movie })
```

Reply:
I created type and sestructured props.